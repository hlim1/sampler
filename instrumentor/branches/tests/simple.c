int main(int argc, char *argv[] __attribute__((unused)))
{
  if (argc > 1)
    return 1;
  else
    return 0;
}
