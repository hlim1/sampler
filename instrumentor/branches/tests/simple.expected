simple.c	3	main	(*((*(argv + 0)) + 0))	0	1
simple.c	6	main	argc > 2	0	0
simple.c	6	main	argc > 1	1	0
